src/components/BooksForm/books-form.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createBook } from '../../actions';
+
+import './books-form.css';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: categories[0],
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleSubmit(e) {
+    const { title, category } = this.state;
+    const { createBook } = this.props;
+    e.preventDefault();
+    createBook({ id: Math.floor(Math.random() * 100).toString(), title, category });
+  }
```

Comment:
When a submission is made you want to reset the target element, so for the text field it should be empty and for the category, it should show a default. This has more to do with the value in state, so look into that